src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -182,6 +183,27 @@ public AndroidDriver(HttpClient.Factory httpClientFactory, Capabilities capabili
         super(httpClientFactory, ensurePlatformName(capabilities, ANDROID_PLATFORM));
     }
 
+    /**
+     * Creates a new instance based on the given ClientConfig and {@code capabilities}.
+     * The HTTP client is default client generated by {@link HttpCommandExecutor#getDefaultClientFactory}.
+     *
+     * Example:
+     *
+     *  ClientConfig clientConfig = ClientConfig.defaultConfig()
+     *    .baseUri(URI.create("WebDriver URL"))
+     *    .proxy(new Proxy(Proxy.Type.HTTP, new InetSocketAddress("your.proxy.ip", 8080)))
```

Comment:
We anyway don't support proxies, so this documentation might be confusing for users

Reply:
same below